charts/Makefile

```diff
@@ -2,35 +2,63 @@ OCP_VERSION ?= 4.19
 BRANCH ?= master
 SYNC2CHARTS ?= true
 ORGREPO ?= https://github.com/openshift
+PROJECTS = \
+	cluster-api \
+	cluster-api-provider-aws \
+	cluster-api-provider-openshift-assisted
 
-build-cluster-api-chart: PROJECT = cluster-api
-build-cluster-api-aws-chart: PROJECT = cluster-api-provider-aws
-build-cluster-api-azure-chart: PROJECT = cluster-api-provider-azure
-
+ifeq ($(BRANCH),master)
+	TAG := latest
+else
+	TAG := $(patsubst release-%, %,$(BRANCH))
```

Comment:
I don't see release branches in [cluster-api-provider-openshift-assisted](https://github.com/openshift-assisted/cluster-api-provider-openshift-assisted). At the time we cherry-pick this work to next release branch [release-2.9](https://github.com/stolostron/cluster-api-installer/blob/release-2.9/charts/Makefile#L2) what branch you will use ?

Reply:
It's yet to be decided, but in this PR I am assuming we will also be using release-2.9.

Reply:
FWIW the format that the MCE team typically expects is `backplane-2.9` .. not sure if we'd need something different though for some reason.

Reply:
the branch release-2.9 for cluster-api-installer is just mirror for backplane-2.9 branch